core/src/test/kotlin/fr/sncf/osrd/pathfinding/PathfindingV2Test.kt

```diff
@@ -96,6 +101,7 @@ class PathfindingV2Test : ApiTest() {
                 .act(RqFake("POST", "/v2/pathfinding/blocks", requestBody))
         val response = TakesUtils.readBodyResponse(rawResponse)
         val parsed =
-            (pathfindingResponseAdapter.fromJson(response) as? IncompatibleElectrification)!!
+            (pathfindingResponseAdapter.fromJson(response)
+                as? IncompatibleConstraintsPathResponse)!!
```

Comment:
You could check right now that the `incompatibleElectrifiations` is equal to the default value you put in the endpoint (that way it'll fail when you actually put the correct values later on).

Reply:
Indeed, I started to improve test on this (amended commit): https://github.com/OpenRailAssociation/osrd/compare/0a2d394be4e8afa67d5252bcab9022c61100a769..727408dc70f9b52c560dcb0e56dcb417f4fc7b0c